packages/ui-library/src/components/input-field-number/index.stories.ts

```diff
@@ -138,18 +141,7 @@ export default {
           suffix: 'suffix',
         },
       },
-      if: { arg: 'hasUnit', eq: true },
-    },
-    unit: {
-      description: 'Select a unit which is displayed next to the input.',
-      options: [undefined, ...Units],
-      control: {
-        type: 'select',
-      },
-      table: {
-        category: 'Content / Settings',
-      },
-      if: { arg: 'hasUnit', eq: true },
+      if: { arg: 'unit', neq: undefined },
```

Comment:
The neq operator stands for "not equal." in Storybook.

Reply:
I think you can completely remove the if statement because by the description it means that the unit will always be present just that the value might be either "prefix or suffix".

Reply:
I changed it so undefined is not a value of the Units array anymore. Units can be undefined.

Reply:
Aah, makes sense, please ignore it then. Still the pipeline is failing, please fix that and should be good.

Reply:
done